Podfile

```diff
@@ -169,7 +169,7 @@ abstract_target 'Apps' do
   ## Gutenberg (React Native)
   ## =====================
   ##
-  gutenberg tag: 'v1.78.1'
+  gutenberg :tag => 'v1.79.0'
```

Comment:
I'm wondering why the tag format has been changed 🤔, should we use `:tag` instead?

Reply:
Looks like this is causing an error: https://buildkite.com/automattic/wordpress-ios/builds/8803#0181ddb1-7b96-4bc2-a51f-7fac80e316cd/572-576

Reply:
I've addressed this issue in https://github.com/wordpress-mobile/WordPress-iOS/pull/19004/commits/73eb00bb4a00daa47ff6fe6bda36447a87da1ba0.